lambda/translattribute.ml

```diff
@@ -638,50 +518,6 @@ let add_poll_attribute expr loc attributes =
       lfunction_with_attr ~attr funct
     end
   | expr -> expr
-||||||| merged common ancestors
-(* Get the [@inlined] attribute payload (or default if not present).
-   It also returns the expression without this attribute. This is
-   used to ensure that this attribute is not misplaced: If it
-   appears on any expression, it is an error, otherwise it would
-   have been removed by this function *)
-let get_and_remove_inlined_attribute e =
-  let attr, exp_attributes =
-    find_attribute is_inlined_attribute e.exp_attributes
-  in
-  let inlined = parse_inline_attribute attr in
-  inlined, { e with exp_attributes }
-=======
-  match expr, get_poll_attribute attributes with
```

Comment:
I'm pretty sure this upstream code behaves the same as our version. Might as well use the upstream version to avoid future conflicts.

Reply:
I think the internally-adopted style is actually necessary for the misplaced attribute warning to work.

See Chris's commit here: https://github.com/ocaml-flambda/ocaml-jst/commit/a9268d29b47cd5c3c051e761f454330573f7ee0e

The conflict will be resolved when he finishes upstreaming that.